run/GCClassic/createRunDir.sh

```diff
@@ -796,6 +824,7 @@ done
 
 mkdir -p ${rundir}
 mkdir -p ${rundir}/Restarts
+ls ${rundir}/Restarts
```

Comment:
Is this line meant to be here or is it leftover from debugging?

Reply:
Now fixed in commit c273374

Reply:
Now fixed in commit 610d00b